egs/csj/s5/local/nnet3/run_tdnn.sh

```diff
@@ -43,7 +43,7 @@ local/nnet3/run_ivector_common.sh --stage $stage \
 gmm_dir=exp/$gmm
 ali_dir=exp/${gmm}_ali_${train_set}_sp
 dir=exp/nnet3${nnet3_affix}/tdnn${affix}
-train_ivector_dir=exp/nnet3${nnet3_affix}ivectors_${train_set}_sp_hires
+train_ivector_dir=exp/nnet3${nnet3_affix}/ivectors_${train_set}_sp_hires
```

Comment:
this looks like a useful change.

Reply:
Sorry, I wrongly made pull-request.
I will re-make pull-request about fixing typo.